### ISSUE TITLE
How can I use it in a remote server?

### ISSUE THREAD

**Issue by IanSmith123** [rofl0r/proxychains-ng#155]
I have a remote server named A(run CentOS), which I want use as a shadowsocks cilent, and I want to use another remote sever named B(run Ubuntu server) as a shadowsocks server. I use ssh to control my remote server A and B. Because of some reasons, I want some flow from sever A go through sever B, so how can I write my proxychains.conf to realize it.  Just like flow
```
socks5 123.45.67.89 1234 rc4-md5 password
```
rc4-md5 is a kind of encrypt.

Thanks anyway!

Les1ie
2017.1.16

**Comment by IanSmith123** [comment]
No.... What stupid guyI am. I can use shadowsocks client in terminal then use 
```
socks5 127.0.0.1 1080
```
Everything goes right.
Thanks anyway.

Les1ie
2017.1.16

**Comment by rofl0r** [comment]
note that you can also use `ssh -D 1080 user@serverB` to create an encrypted ssh tunnel to server B and spawn a local socks5 proxy listening on port 1080 where you can connect your stuff to via proxychains.

**Comment by rofl0r** [comment]
btw, i have an application that's intended to keep an ssh-socks5-tunnel alive by doing continuous connectivity checks, since ssh connections tend to get stuck after a while:
https://github.com/rofl0r/rocksock/blob/master/examples/ssh-socks-restart.c
